docs-source/spring/content/development/project.md

```diff
@@ -168,38 +170,56 @@ spring:
 ```
 
 The variables in this configuration are automatically injected to your deployment
-and pods when you use the Oracle Backend for Spring Boot and Microservices CLI to deploy your application. 
+and pods when you use the Oracle Backend for Spring Boot and Microservices CLI to deploy your application.
 When you use the `deploy` command, you must specify the `liquibase-db` parameter and
-provide a user with sufficient privileges, generally this will be premissions to create and alter
+provide a user with sufficient privileges, generally this will be permissions to create and alter
```

Comment:
Change to "... sufficient privileges. Generally, this will be permission to create ..."